SwiftLeeds/Views/Local/BottomSheetView.swift

```diff
@@ -10,6 +10,9 @@ import SwiftUI
 struct BottomSheetView: View {
     @GestureState private var translation: CGFloat = 0
     @Binding var isOpen: Bool
+
+    let categories: [Local.LocationCategory]
+    let error: Error?
```

Comment:
I couldn't see an existing approach for error handling so kept it really basic for now as to leave It open for whoever else wants to do it